gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1041,8 +1052,12 @@ function PrepareRound()
 	-- Update damage scaling
 	KARMA.RoundPrepare()
 
-	-- New look. Random if no forced model set.
-	GAMEMODE.playermodel = GAMEMODE.force_plymodel == "" and GetRandomPlayerModel() or GAMEMODE.force_plymodel
+	-- New look. Random if no forced model set
+	if cvPreferMapModels:GetBool() and GAMEMODE.force_plymodel and GAMEMODE.force_plymodel ~= "" then
```

Comment:
```suggestion
	if cvPreferMapModels:GetBool() and GAMEMODE.force_plymodel ~= "" then
```

Reply:
Im actually unsure about this suggestion, is nil already excluded? Might need to check this